src/annotations.h

```diff
@@ -0,0 +1,25 @@
+/*
+ * This file and its contents are licensed under the Apache License 2.0.
+ * Please see the included NOTICE for copyright information and
+ * LICENSE-APACHE for a copy of the license.
+ */
+#ifndef TIMESCALEDB_ANNOTATIONS_H
+#define TIMESCALEDB_ANNOTATIONS_H
+
+/* Fall-through annotation */
+#if defined(__clang__)
+#if (__clang_major__ >= 12) || (__clang_analyzer__)
+#define TS_FALLTHROUGH __attribute__((fallthrough))
```

Comment:
Just to note that it is also possible to write /`* fallthough */` comment which will be recognized, maybe it could be easier to support in some sense

Reply:
This was the issue. It seems to no longer be supported in clang 12 (at  least on the mac)